sidekiq_publisher.gemspec

```diff
@@ -58,5 +58,5 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "activerecord-postgres_pub_sub", ">= 0.4.0"
   spec.add_runtime_dependency "activesupport", ">= 5.1", "< 6.1"
-  spec.add_runtime_dependency "sidekiq", ">= 5.0.4", "< 6.1"
+  spec.add_runtime_dependency "sidekiq", ">= 5.0.4", "< 6.2"
```

Comment:
For Sidekiq I'd be okay with extending this to "< 7.0" but don't feel strongly.

Reply:
Works for me! Just updated this to be `< 7.0`.